dev-vcs/qgit/qgit-2.9.recipe

```diff
@@ -0,0 +1,60 @@
+SUMMARY="A Qt-based git GUI viewer"
+DESCRIPTION="With qgit you will be able to browse revisions history,\
+view patch content and changed files, graphically following different \
+development branches."
+HOMEPAGE="https://github.com/tibirna/qgit"
+COPYRIGHT="2011-2019 Cristian Tibirna"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/tibirna/qgit/archive/qgit-$portVersion.tar.gz"
+CHECKSUM_SHA256="69788efda7d434d1bc094bb414bd92c269dc7894326320634500b05d63c008e8"
+SOURCE_DIR="qgit-qgit-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	qgit$secondaryArchSuffix = $portVersion
+	cmd:qgit
+	app:QGit
+	"
+REQUIRES="
+	haiku
+	cmd:git
+	lib:libGL$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Widgets$secondaryArchSuffix
+	qthaikuplugins$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Widgets$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:qmake$secondaryArchSuffix >= 5
+	"
+
+defineDebugInfoPackage qgit$secondaryArchSuffix \
+	$appsDir/QGit
+
+BUILD()
+{
+	qmake qgit.pro
+	make $jobArgs
+}
+
+INSTALL()
+{
+	mkdir -p $appsDir
+	mkdir -p $binDir
+	cp bin/qgit $appsDir
+	mv $appsDir/qgit $appsDir/QGit
+	cp $appsDir/QGit $binDir/qgit
```

Comment:
Should be a symlink instead of a copy.

Reply:
I've tried ln -s $appsDir/QGit $binDir/qgit but it says that the target does not exist.
How should I fix this?

Reply:
You mean the target already exists?

Reply:
No. The target **does not** exist.

Reply:
@korli This time, I cleaned the whole work directory, and built it from scratch, and the symlink worked.